src/Web/Encoding/TextDecoder.js

```diff
@@ -1,11 +1,9 @@
-"use strict";
-
-exports.new = function(label) {
+export function new(label) {
```

Comment:
Oops — this needs to be fixed, it's reserved.

Reply:
Thanks!